src/utils/constants.js

```diff
@@ -0,0 +1,10 @@
+export const SORTED_BY = {
+  DEFAULT: '',
+  NAME: 'name',
+  LENGTH: 'length',
+};
+
+export const REVERS_STATUS = {
+  REVERSED: 'reversed',
+  NOT_REVERSED: 'not reversed',
+};
```

Comment:
This looks like an overkill

Probably better just use boolean `isReversed` here

But it is not critical